backend/migrations/versions/5fc49baecea4_add_enviromental_parameters.py

```diff
@@ -31,7 +31,7 @@
 
```

Comment:
We need to adjust the revision numbers. Right now we have:
add_environments: 5fc49baecea4, DOWN=`509997f0a51e`
share_request: 72b8a90b6ee8,  DOWN=`509997f0a51e`
rename_imported: e1cd4927482b, DOWN= 72b8a90b6ee8

And we should have
share_request: 72b8a90b6ee8,  DOWN=509997f0a51e
rename_imported: e1cd4927482b, DOWN= 72b8a90b6ee8
add_environments: 5fc49baecea4, DOWN=`e1cd4927482b`